tests/test_integration.py

```diff
@@ -770,6 +770,15 @@ async def test_hoprd_check_ticket_price_is_default(peer, swarm7: dict[str, Node]
     assert price > 0
 
 
+@pytest.mark.asyncio
+@pytest.mark.parametrize("peer", random.sample(barebone_nodes(), 1))
+async def test_hoprd_check_ticket_winn_prob_is_default(peer, swarm7: dict[str, Node]):
+    price = await swarm7[peer].api.ticket_winn_prob()
+
+    assert isinstance(price, float)
+    assert price > 0
```

Comment:
```suggestion
    assert 0.0 <= pytest.approx(price) <= 1.0
```

Reply:
This cannot work. `approx()` method only works for equality checks, not for `gt`, `ge`, `le`, `lt`. Using `round(..., 5)` instead.